dev/packages/beats/activemq-0.0.1/dataset/audit/elasticsearch/ingest-pipeline/pipeline.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
if the pipeline is called `default.yml` or `default.json` we automatically fall back to it without any additional config in the manifest. Perhaps worth renaming it.

Reply:
Let's consider these two cases:

1. Logstash

```
../beats/filebeat/module/logstash/slowlog/ingest/pipeline-json.yml
../beats/filebeat/module/logstash/slowlog/ingest/pipeline-plain.yml
```
Which one should called `default.json`? Should I rename it only if there is a single pipeline?

2. Elasticsearch

```
../beats/filebeat/module/elasticsearch/slowlog/ingest/pipeline-json.yml
../beats/filebeat/module/elasticsearch/slowlog/ingest/pipeline-plaintext.yml
../beats/filebeat/module/elasticsearch/slowlog/ingest/pipeline.yml
```

Which one should be called `default.yml`?

Reply:
In the case of Elasticsearch, it is the `pipeline.yml` file as this is the one doing the routing. I had to dig into the LS one to see why it even works. It turns out this is an anti pattern from moving forward. It decides the ingest pipeline based on the config. Instead there should always be 1 pipeline. The LS module should be modified to use the ES pattern, otherwise it will not work with the packages.

@ph @ycombinator @exekias We should change the LS module.

Reply:
Yes, this has been on the cards for a while anyway: https://github.com/elastic/beats/issues/9964.

Reply:
Fixed.

Reply:
> Fixed.

Hey @mtojek, just to clarify, we still need elastic/beats#9964 to remain open, right?

Reply:
> > Fixed.
> 
> Hey @mtojek, just to clarify, we still need [elastic/beats#9964](https://github.com/elastic/beats/issues/9964) to remain open, right?

Yes, would be great to fix this.

Reply:
@ycombinator Yes, it now even becomes a must to change to migrate to packages.

Reply:
Thanks for clarifying. @ruflin other than "ASAP" 😄 what's the timeline you need this to be fixed in?